js/ui/tab_panel.js

```diff
@@ -22,6 +22,8 @@ const TABPANEL_CONTAINER_CLASS = 'dx-tabpanel-container';
 
 const TABS_ITEM_TEXT_CLASS = 'dx-tab-text';
 
+const FOCUS_ON_DISABLED_CLASS = 'dx-disabled-tab-focused';
```

Comment:
Finally I think `dx-disabled-focused-tab` is more suitable, sorry :)
Let's also update all an appropriate code - class name, toggle methods name and so on
```suggestion
const DISABLED_FOCUSED_TAB_CLASS = 'dx-disabled-focused-tab';
```

Reply:
Good name! Used in new version: [1254ebd2f865395198a8f28647dccaf61766bada].